src/js/controllers/mapController.ts

```diff
@@ -986,6 +986,23 @@ export class MapController {
     }
   }
 
+  async getDocuments(
+    urlProperties: URLProperties
+  ): Promise<Array<Attachment> | null> {
+    const { iso, layerTitle, sublayerID, specificFeatureID } = urlProperties;
+    const endPoint = `https://gis.forest-atlas.org/server/rest/services/${iso}/${layerTitle}/MapServer/${sublayerID}/${specificFeatureID}/attachments?f=pjson`;
```

Comment:
What is the reason we are not using `queryAttachments()` built in ESRI method on FeatureLayers? https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-FeatureLayer.html#queryAttachments
With this we would not need to keep this dynamic URL construction and just hit FeatureLayer endpoint and pass down object id that we care about? Can we check if that approach would return results?